### ISSUE TITLE
adapt-contrib-trickle: Button focus sometimes goes missing after click

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2518]
### Subject of the issue
When clicking on the 'continue' button the screen reader cursor occasionally jumps backward before moving forward causing the previous component to read before the scroll is finished and focus is set to the next component.

### Steps to reproduce
Trickle a text component onto an mcq.

### Expected behaviour
Focus should be set before the scroll is triggered, on the button click, after trickle has resized and has exposed the next component.
